index.js

```diff
@@ -3,20 +3,16 @@ const fs = require('node:fs');
 const path = require('node:path');
 const { Client, Collection, Events, GatewayIntentBits} = require('discord.js');
 const { token, channelId } = require('./config.json');
-const axios = require('axios');
```

Comment:
deleted unnecessary require lines to clean up index.js file

Reply:
just a btw in case ur wondering why i did this, feel free to resolve this once you've seen it